Client/Ecosia/UI/PageAction/PageActionMenuCell.swift

```diff
@@ -218,8 +219,8 @@ extension PageActionMenuCell {
                 badgeLabel.text = .localized(.new)
                 badge.addSubview(badgeLabel)
                 
-                badgeLabel.topAnchor.constraint(equalTo: badge.topAnchor, constant: 2.5).isActive = true
-                badgeLabel.leftAnchor.constraint(equalTo: badge.leftAnchor, constant: 8).isActive = true
```

Comment:
Wouldn't removing horizontal limits result in a potential overflow, if the text in the badge is very long?